packages/frontend/src/content/ShowNote.tsx

```diff
@@ -27,21 +28,22 @@ const ShowNote = (props: RouteComponentProps<{ noteId: string }>) => {
           setAttachmentURL(await getObjectUrl(data.attachment));
         }
       } catch (error) {
+        console.error(error);
+        setErrorMsg("Failed to load note. Redirecting...");
```

Comment:
Should these messages be removed?
```suggestion
```

The errorMsg is supposed to be display if a specific error has happened, and no action is taken by the app. This won't have any action since we navigate to 404 page.

Reply:
Yes correct, Since the 404 page handles the error messaging for the user, and we’re immediately redirecting, both setErrorMsg and console.error(error) are unnecessary. I’ll remove them.